src/main/java/org/csanchez/jenkins/plugins/kubernetes/pod/yaml/Overrides.java

```diff
@@ -0,0 +1,40 @@
+package org.csanchez.jenkins.plugins.kubernetes.pod.yaml;
+
+import hudson.Extension;
+import hudson.model.Descriptor;
+import io.fabric8.kubernetes.api.model.Pod;
+import org.jenkinsci.Symbol;
+import org.kohsuke.stapler.DataBoundConstructor;
+
+import javax.annotation.Nonnull;
+import java.io.Serializable;
+import java.util.List;
+
+import static org.csanchez.jenkins.plugins.kubernetes.PodTemplateUtils.parseFromYaml;
+
+public class Overrides extends YamlMergeStrategy implements Serializable {
```

Comment:
Why is it `Serializable`? And if it needs to be, then should this be declared on the supertype?